### PR TITLE
Add temporary custom post types Person and Program.

### PR DESCRIPTION
**Pull request by gtenaschuk**
- Add custom post types "Person" and "Program" to be removed after the import is complete in live environment.

### CODE REVIEW

File: wp-content/plugins/tw-import-post-types/tw-import-post-types.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		"public" => true,
+		"publicly_queryable" => true,
+		"show_ui" => true,
+		"show_in_rest" => false,
+		"rest_base" => "",
+		"rest_controller_class" => "WP_REST_Posts_Controller",
+		"rest_namespace" => "wp/v2",
+		"has_archive" => true,
+		"show_in_menu" => true,
+		"show_in_nav_menus" => true,
```
Comment (rpeterman-gp) [inline wp-content/plugins/tw-import-post-types/tw-import-post-types.php:42]: Let's set this to false for both post types.

File: wp-content/plugins/tw-import-post-types/tw-import-post-types.php [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		"delete_with_user" => false,
+		"exclude_from_search" => false,
+		"capability_type" => "post",
+		"map_meta_cap" => true,
+		"hierarchical" => false,
+		"can_export" => true,
+		"rewrite" => [ "slug" => "person", "with_front" => true ],
```
Comment (rpeterman-gp) [inline wp-content/plugins/tw-import-post-types/tw-import-post-types.php:49]: Can we change these slugs to avoid collision with the taxonomies' slugs?

**Review by rpeterman-gp — CHANGES_REQUESTED** [review]
A couple changes to prevent these post types from ending up in menus and their slugs matching the taxonomies' slugs.

File: wp-content/plugins/tw-import-post-types/tw-import-post-types.php (hunk 2 above)
Comment (gtenaschuk) [reply wp-content/plugins/tw-import-post-types/tw-import-post-types.php:49]: Yes we can, i just made a copy/paste from what we had in CPT UI.

File: wp-content/plugins/tw-import-post-types/tw-import-post-types.php (hunk 1 above)
Comment (gtenaschuk) [reply wp-content/plugins/tw-import-post-types/tw-import-post-types.php:42]: Yup, will do.

Comment (gtenaschuk) [reply wp-content/plugins/tw-import-post-types/tw-import-post-types.php]: Done @rpeterman-gp I also disabled the archive list and excluded this post type from the front end search.

File: wp-content/plugins/tw-import-post-types/tw-import-post-types.php (hunk 2 above)
Comment (gtenaschuk) [reply wp-content/plugins/tw-import-post-types/tw-import-post-types.php]: Done @rpeterman-gp , i added a "-post" as suffix

**Comment by gtenaschuk** [comment]
@rpeterman-gp please let me know when you check this and if there are more adjustments you want to do. Thanks

**Review by rpeterman-gp — APPROVED** [review]
Thanks. this looks good. :shipit:

**Pull request merged**